boards/bluepill/include/periph_conf.h

```diff
@@ -152,6 +152,41 @@ static const uart_conf_t uart_config[] = {
 #define UART_NUMOF          (sizeof(uart_config) / sizeof(uart_config[0]))
 /** @} */
 
+/**
+ * @name I2C configuration
+ * @{
+ */
+#define I2C_NUMOF           (2U)
+#define I2C_0_EN            1
+#define I2C_1_EN            0 /* Disabled by default since Pins collide with USART3. */
```

Comment:
very nit-picky: /Pins/pins/

Reply:
if addressed, I promise to merge right away 😉

Reply:
(iff murdock happy)

Reply:
oops, sure will fix it right away